pallets/system/benchmarking/src/lib.rs

```diff
@@ -144,7 +144,7 @@ benchmarks! {
 		let block_length = Default::default();
 
 	}: {
-		let _header = T::HeaderExtensionBuilder::build(app_extrinsics, data_root, block_length);
+		let _header = T::HeaderExtensionBuilder::build(app_extrinsics, data_root, block_length, 0);
```

Comment:
Is the last param seed? What does seed `0` mean?

Reply:
No, it's not. That parameter is the `block_number` and that is not used in production yet.
I only use it during the corruption header test (enabled by `header_commitment_corruption` feature), where the bad behaviour starts after block `20`.